app/timetables/models.py

```diff
@@ -1,3 +1,12 @@
 from __future__ import unicode_literals
 
 from django.db import models
+
+from app.shared.models import CommonInfo
+
+
+class Weekday(CommonInfo):
```

Comment:
Hi @andela-kanyanwu I know I've already 👍 but let's test this model in the `timetables` app
Sample testing folder structure we could follow:
https://realpython.com/blog/python/testing-in-django-part-1-best-practices-and-examples/